lmdeploy/tokenizer.py

```diff
@@ -127,9 +127,9 @@ def __init__(self, model_dir: str):
             if osp.exists(generation_config_file):
                 with open(generation_config_file, 'r') as f:
                     cfg = json.load(f)
-                    self.model.eos_token_id = cfg['eos_token_id']
+                    self.eos_token_id_ = self.model.eos_token_id = cfg['eos_token_id']
             elif hasattr(self.model, 'eod_id'):  # Qwen remote
-                self.model.eos_token_id = self.model.eod_id
+                self.eos_token_id_ = self.model.eos_token_id = self.model.eod_id
```

Comment:
Sorry. But, what does this fix by assign another value?

Reply:
In HuggingFace self.model.eos_token_id is not a attritube and cannot be assigned a value. You can try it by:
```
from lmdeploy.tokenizer import Tokenizer
import os.path as osp

tokenizer_model_path = osp.join(model_path, 'triton_models',
                                        'tokenizer')
tokenizer = Tokenizer(tokenizer_model_path)
tokenizer.eos_token_id = 151643
print(tokenizer.eos_token_id)
```

self.eos_token_id_ can be a alternative to return if self.model.eod_id is not None

Reply:
I see.